index.template.html

```diff
@@ -4723,7 +4725,42 @@ <h3>Extension and Import</h3>
         }
     </pre>
 
+    <p>
+        As an example for relative imports using <code>tm:ref</code>, the 
+        following Thing Model re-uses and augments (see below) a 
+        <code>genericTemperature</code> property in two more specific 
+        properties, which describe an inner and an outer temperature value, 
+        respectively.
+    </p>
 
+    <pre class="example" title="Multi Sensor Thing Model re-using definitions with relative imports" id="td-model-example-relative-tmRef">
+        {
+            "@context": "https://www.w3.org/2022/wot/td/v1.1", 
+            "@type" : "tm:ThingModel",
+            "title": "Multi Sensor",
+            "properties": {
+            "genericTemperature": {
+                "type": "number",
+                "unit": "C"
+            },
+            "innerTemperature": {
+                "tm:ref": "#/properties/genericTemperature",
+                "title": "The inner temperature",
+                "minimum": 10
+            },
+            "outerTemperature": {
+                "tm:ref": "#/properties/genericTemperature",
+                "title": "The outer temperature",
+                "description": "The outer temperature is measured in Kelvin",
+                "unit": "K"
+            }
```

Comment:
The JSON intendation of the properties should be one level deeper here, improving readbility.

Reply:
Thank you! Should be fixed now :)